setup.cfg

```diff
@@ -66,3 +66,13 @@ ignore =
     W391
 exclude = pyramid/tests/,pyramid/compat.py,pyramid/resource.py
 show-source = True
+
+[check-manifest]
+ignore =
+    .gitignore
+    PKG-INFO
+    *.egg-info
+    *.egg-info/*
+ignore-default-rules = true
+ignore-bad-ideas =
```

Comment:
Just FYI the bad-ideas are errors that check-manifest outputs. Here is the failure without it:

```
lists of files in version control and sdist match
you have pyramid/tests/pkgs/localeapp/locale/de/LC_MESSAGES/deformsite.mo in source control!
that's a bad idea: auto-generated files should not be versioned
this also applies to the following:
  pyramid/tests/pkgs/localeapp/locale/de_DE/LC_MESSAGES/deformsite.mo
  pyramid/tests/pkgs/localeapp/locale/en/LC_MESSAGES/deformsite.mo
  pyramid/tests/pkgs/localeapp/locale2/de/LC_MESSAGES/deformsite.mo
  pyramid/tests/pkgs/localeapp/locale2/en/LC_MESSAGES/deformsite.mo
  pyramid/tests/pkgs/localeapp/locale3/de/LC_MESSAGES/deformsite.mo
  pyramid/tests/pkgs/localeapp/locale3/en/LC_MESSAGES/deformsite.mo
```